packages/queries/src/types/QueryRequest.ts

```diff
@@ -8,10 +8,15 @@ export interface RegularQueryRequest {
     query: AnyQuery;
 }
 
-export interface RegisteredQueryRequest {
-    type: typeof RegisteredQuery;
+export interface RegisteredQueryRequestBody {
     queryId: string;
     params: Record<string, unknown>;
 }
 
+export interface RegisteredQueryRequest {
```

Comment:
why do you need RegisteredQueryRequest and RegisteredQueryRequestBody?

Reply:
`RegisteredQueryRequestBody` is used to type `executeRegisteredQuery()`